<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <strikeout>false</strikeout>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="loginGroup">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>371</width>
      <height>371</height>
     </rect>
    </property>
    <property name="title">
     <string>Sign In</string>
    </property>
    <widget class="QPushButton" name="pushButtonLogin">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>90</y>
       <width>261</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelUsername">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditUsername">
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelPassword">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="font">
           <font>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
          </property>
          <property name="inputMask">
           <string notr="true"/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButtonCreateAccount">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Create Account</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="libGroup">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1341</width>
      <height>731</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="title">
     <string>Online Library</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1321</width>
       <height>701</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="LibraryLayout">
      <item>
       <widget class="QGroupBox" name="filterGroup">
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Filter</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="layoutPage">
           <item>
            <widget class="QLabel" name="labelPage">
             <property name="text">
              <string>Page</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxPage">
             <property name="editable">
              <bool>true</bool>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="maxCount">
              <number>50</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="layoutItemsPage">
           <item>
            <widget class="QLabel" name="labelItemsPage">
             <property name="text">
              <string>Entries per page</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBoxItemsPage">
             <property name="editable">
              <bool>false</bool>
             </property>
             <property name="currentText">
              <string/>
             </property>
             <property name="maxVisibleItems">
              <number>3</number>
             </property>
             <property name="maxCount">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="layoutAuthorTitle">
           <item>
            <widget class="QLabel" name="labelTitle">
             <property name="text">
              <string>Book Title</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditTitle"/>
           </item>
           <item>
            <layout class="QVBoxLayout" name="layoutAuthor">
             <item>
              <widget class="QLabel" name="labelAuthor">
               <property name="text">
                <string>Author Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditAuthor"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutYear">
           <item>
            <layout class="QVBoxLayout" name="layoutYearLabels">
             <item>
              <widget class="QLabel" name="labelMinYear">
               <property name="text">
                <string>After</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelMaxYear">
               <property name="text">
                <string>Before</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="layoutYearSliders">
             <item>
              <widget class="QSlider" name="sliderMinYear">
               <property name="minimum">
                <number>1800</number>
               </property>
               <property name="maximum">
                <number>2050</number>
               </property>
               <property name="value">
                <number>1800</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedControls">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sliderMaxYear">
               <property name="minimum">
                <number>1800</number>
               </property>
               <property name="maximum">
                <number>2050</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedControls">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QDateEdit" name="dateEditMin">
               <property name="maximumDate">
                <date>
                 <year>2050</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
               <property name="minimumDate">
                <date>
                 <year>1800</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
               <property name="maximumTime">
                <time>
                 <hour>23</hour>
                 <minute>59</minute>
                 <second>59</second>
                </time>
               </property>
               <property name="displayFormat">
                <string>yyyy</string>
               </property>
               <property name="calendarPopup">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateEdit" name="dateEditMax">
               <property name="maximumDateTime">
                <datetime>
                 <hour>23</hour>
                 <minute>59</minute>
                 <second>59</second>
                 <year>2050</year>
                 <month>1</month>
                 <day>1</day>
                </datetime>
               </property>
               <property name="minimumDate">
                <date>
                 <year>1800</year>
                 <month>1</month>
                 <day>1</day>
                </date>
               </property>
               <property name="displayFormat">
                <string>yyyy</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layoutRating">
           <item>
            <widget class="QLabel" name="labelRating">
             <property name="text">
              <string>Rating</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBoxRating"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="labelGenre">
           <property name="text">
            <string>Genre</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollAreaGenre">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaGenreContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>617</width>
              <height>331</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QCheckBox" name="checkBoxDrama">
               <property name="text">
                <string>Drama</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxComedy">
               <property name="text">
                <string>Comedy</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxThriller">
               <property name="text">
                <string>Thriller</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxHorror">
               <property name="text">
                <string>Horror</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxManual">
               <property name="text">
                <string>Manual</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxBio">
               <property name="text">
                <string>Biography</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxAutobio">
               <property name="text">
                <string>Autobiography</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxChildren">
               <property name="text">
                <string>Children</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxAction">
               <property name="text">
                <string>Action</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxRomance">
               <property name="text">
                <string>Romance</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxFantasy">
               <property name="text">
                <string>Fantasy</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonApplyFilter">
           <property name="text">
            <string>Filter Results</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="getRecommendedBooks">
           <property name="text">
            <string>Recommend Books</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidgetQueryResult">
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="verticalScrollMode">
         <enum>QAbstractItemView::ScrollPerItem</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>8</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>154</number>
        </attribute>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <zorder>libGroup</zorder>
   <zorder>loginGroup</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClient">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionHide_Show"/>
   </widget>
   <addaction name="menuClient"/>
   <addaction name="menuFilter"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionHide_Show">
   <property name="text">
    <string>Hide/Show</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
